frontend/src/views/CensusDirectory/components/Project/ProjectButtons/components/EmbeddingButton/connect.ts

```diff
@@ -28,25 +25,46 @@ function pythonCodeSnippet(project: StaticProject | Project): string {
         census,
         organism = "${organism}",
         measurement_name = "${measurement}",
-        obs_value_filter = "tissue == 'tongue'",
-        obsm_layers = "${project.obs_matrix}"
+        obs_value_filter = "tissue_general == 'central nervous system'",
+        obsm_layers = ["${project.obsm_layer}"]
     )`
     : `  import cellxgene_census
   from cellxgene_census.experimental import get_embedding
 
-  embedding_uri = ${uri}
+  embedding_uri = \\
+      "${uri}"
   census = cellxgene_census.open_soma(census_version="${censusVersion}")
 
   adata = cellxgene_census.get_anndata(
       census,
       organism = "${organism}",
       measurement_name = "${measurement}",
-      obs_value_filter = "tissue == 'tongue'",
+      obs_value_filter = "tissue_general == 'central nervous system'",
   )
-  embeddings = get_embedding("${censusVersion}", embedding_uri, adata.obs["soma_joinid"])
+  embeddings = get_embedding("${censusVersion}", embedding_uri, adata.obs["soma_joinid"]).tonumpy())
   adata.obsm["emb"] = embeddings`;
 }
 
+function rCodeSnippet(project: UnionProject): string {
+  const censusVersion = project.census_version;
+  const organism = project.experiment_name;
+
+  return project.tier === "maintained"
+    ? `    library("cellxgene.census")
+    library("Seurat")
+
+    census <- open_soma(census_version = "${censusVersion}")
+    seurat_obj <- get_seurat(
+      census,
+      organism = "${organism}",
+      obs_value_filter = "tissue_general == 'central nervous system'",
+      obs_column_names = c("cell_type"),
+      obsm_layers = c("${project.obsm_layer}")
+    )
+    `
+    : "";
```

Comment:
nit: Seems like a guard clause is cleaner if we're returning an empty string?

E.g.,

```ts
if (project.tier !== "maintained") return ""

return ...
```

Reply:
it'll be changed sometime in the near future